interface-definitions/service_ssh.xml.in

```diff
@@ -281,6 +281,25 @@
             </properties>
             <children>
               #include <include/pki/ca-certificate.xml.i>
+              <tagNode name="bind-user">
+                <properties>
+                  <help>user-name</help>
```

Comment:
In order to make this feature properly work this most likely corresponds to a username defined already under: `set system login user` - so this node misses a completionHelper for lazy people like me:

```suggestion
                  <help>Username for principal list</help>
                  <completionHelp>
                    <path>system login user</path>
                  </completionHelp>                  
```

Reply:
I agree, @c-po's idea is better than adding it to `service ssh`.